lib/ansible/constants.py

```diff
@@ -164,3 +165,28 @@ def __getitem__(self, y):
 
 for warn in config.WARNINGS:
     _warning(warn)
+
+
+# The following are hard-coded action names
+_COMMAND_SHELL_SCRIPT_ACTIONS = add_internal_fqcns(('command', 'shell', 'script'))
+_COMMAND_SHELL_SCRIPT_RAW_ACTIONS = add_internal_fqcns(('command', 'shell', 'script', 'raw'))
```

Comment:
same as above _ACTION_ALLOWS_RAW_ARGS .. also can be built better by leveraging existing vs repeating the work
```
_ACTION_ALLOWS_RAW_ARGS = _ACTION_HAS_CMD_ARG + add_internal_fqcns(('raw'))
```